example/src/index.ts

```diff
@@ -71,18 +71,18 @@ function main(): void {
     sequence: ['Enter'],
     selector: '.jp-DirListing',
     handler: () => {
-      fbWidget.rename();
+      fbWidget.open();
       return true;
     }
   }, {
-    sequence: ['ArrowDown'],
+    sequence: ['Ctrl N'],
     selector: '.jp-DirListing',
     handler: () => {
       fbWidget.selectNext();
       return true;
     }
   }, {
-    sequence: ['ArrowUp'],
+    sequence: ['Ctrl P'],
```

Comment:
Yes, this augments the built in shortcuts with the emacs navigation shortcuts for the purposes of the example.